src/test/resources/database/book/add-books-table.sql

```diff
@@ -0,0 +1,8 @@
+insert into books (id, title, author, isbn, price, description, cover_image)
+values (1,"New_man", "Roman", "333", 95.99, "nice book for old people", "https://i.pinimg.com/originals/9f/a8/9e/9fa89e944234f7c3fe04bacc6da638b2.png");
+
+insert into books (id, title, author, isbn, price, description, cover_image)
+values (2,"Super_man", "Andrew", "3323", 150, "nice book for old people", "https://i.pinimg.com/originals/9f/a8/9e/9fa89e944234f7c3fe04bacc6da638b2.png");
+
+insert into books (id, title, author, isbn, price, description, cover_image)
+values (3,"Older_man_in_sea", "Katerina", "3313", 300, "nice book for old people", "https://i.pinimg.com/originals/9f/a8/9e/9fa89e944234f7c3fe04bacc6da638b2.png");
```

Comment:
description and cover_image are not required fields (minor comment)
```suggestion
insert into books (id, title, author, isbn, price, description, cover_image)
values (1,"New_man", "Roman", "333", 95.99, "nice book for old people", "https://i.pinimg.com/originals/9f/a8/9e/9fa89e944234f7c3fe04bacc6da638b2.png"), 
(2,"Super_man", "Andrew", "3323", 150, "nice book for old people", "https://i.pinimg.com/originals/9f/a8/9e/9fa89e944234f7c3fe04bacc6da638b2.png"),
(3,"Older_man_in_sea", "Katerina", "3313", 300, "nice book for old people", "https://i.pinimg.com/originals/9f/a8/9e/9fa89e944234f7c3fe04bacc6da638b2.png");
```